packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -541,6 +544,57 @@ void main() {
       );
     });
 
+    testWidgets('picker persists am/pm value when scrolling hours', (WidgetTester tester) async {
+      DateTime date;
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: SizedBox(
+            height: 400.0,
+            width: 400.0,
+            child: CupertinoDatePicker(
+              mode: CupertinoDatePickerMode.time,
+              onDateTimeChanged: (DateTime newDate) {
+                date = newDate;
+              },
+              initialDateTime: DateTime(2019, 1, 1, 3),
+            ),
+          ),
+        ),
+      );
+
+      // 3:00 -> 15:00
+
```

Comment:
Remove the white space between the comment and the code block that's doing the change the comment is indicating. Otherwise, I wasn't too sure if it's the code above or below doing the change.

Reply:
Done